bitrise.yml

```diff
@@ -1,57 +1,69 @@
 ---
-format_version: "11"
+format_version: '11'
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: react-native
 
 trigger_map:
 - push_branch: "*"
-  workflow: primary
+  pipeline: primary
 - pull_request_source_branch: "*"
-  workflow: primary
-
-workflows:
+  pipeline: primary
 
+pipelines:
   primary:
     description: Runs JavaScript tests, and verifies iOS and Android builds
-    before_run:
-    - _setup
-    after_run:
-    - _test-js
-    - _build-ios
-    - _build-android
+    stages:
+    - test-and-build-stage: {}
+
+stages:
+  test-and-build-stage:
+    abort_on_fail: true
+    workflows:
+    - test-js: {}
+    - build-ios: {}
+    - build-android: {}
 
+workflows:
   _setup:
     steps:
-    - git-clone@6: { }
-    - restore-npm-cache@1: { }
+    - git-clone@6: {}
+    - restore-npm-cache@1: {}
     - npm@1:
         inputs:
         - command: install
-    - save-npm-cache@1: { }
-
-  _test-js:
+    - save-npm-cache@1: {}
+  test-js:
+    before_run:
+    - _setup
     steps:
     - npm@1:
         inputs:
         - command: test
-
-  _build-ios:
+  build-ios:
+    before_run:
+    - _setup
     description: Builds the iOS version of the app for use in Simulators
     steps:
     - react-native-bundle@1:
         inputs:
         - entry_file: index.js
         - platform: ios
-    - restore-cocoapods-cache@1: { }
-    - cocoapods-install@2: { }
-    - save-cocoapods-cache@1: { }
+    - restore-cocoapods-cache@1: {}
+    - script:
```

Comment:
I'd add a title and a description to this step, explaining what magic this performs and why

Reply:
Ended up removing this script and modifying the Podfile directly per the Flipper documentation: 95612bb